### PR TITLE
CITE Participation Guide

### PR DESCRIPTION
**Pull request by weirdscience**
This PR fixes #385 and #389 to host the CITE Participation Guide on the FPKI Guides website. This document was revised based on feedback from a Technical Working Group meeting and a three week community comment period.

### CODE REVIEW

**Comment by lachellel** [comment]
@weirdscience I'm going to add some embedded comments to address; 

- Plain Language and active voice
- Formatting
- Alignment with the other pages (linking, headers)
- Content

I'll try to do a PR too to help address some of the above!

File: _tools/cite-guide.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
+---
+layout: default
+title: CITE Participation Guide
+collection: tools
+permalink: /tools/citeguide/
+---
+
+# Community Interoperability Test Environment (CITE) Participation Guidance
```
Comment (lachellel) [inline _tools/cite-guide.md]: Remove this header - in the templates, the title is displayed first and having an additional top level header creates **two** headers for the page

File: _tools/cite-guide.md [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+1. Ensure proper functionality of system changes prior to production deployment, and
+2. Identify and resolve technical issues across FPKI Partners.
+
+The FPKI Community can use CITE to evaluate PKI or application changes in a test environment that mimics the production FPKI hierarchy and test potential interoperability issues before those changes are deployed to the FPKI. <br>
+
+## Document Scope
+
+This document provides guidelines and specifications for an FPKI Partner to participate in CITE. CITE Participants refer to an FPKI Partner establishing a test PKI certified or cross-certified with the Test Common Policy or Test Bridge CA. This document shall not define how CITE Participants conduct testing.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
```
Comment (lachellel) [inline _tools/cite-guide.md]: Unnecessary - this is not a Standard or a policy doc.  It's a guide.  Align words with active voice.

File: _tools/cite-guide.md [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+
+Modifications to this document and any waivers are controlled by the FPKI Technical Working Group. <br>
```
Comment (lachellel) [inline _tools/cite-guide.md]: What is a waiver in this context?  is there a compliance mechanism that requires a waiver to be granted for auditing purposes or other?

File: _tools/cite-guide.md [hunk 4]
```diff
@@ -0,0 +1,459 @@
...
+Some examples of testing conducted in CITE include:
+1. Interoperability testing between cross-certified Certification Authorities (CAs);
+2. Transition testing to new algorithms (e.g., SHA-2, ECC);
+3. PIV and PIV-I card interoperability testing;
+4. Repository access testing when using content delivery networks, load balancers, or other networking configurations; and
+5. Path discovery and/or validation testing for an application
+
+When testing is successful in CITE, assurance is gained that the proposed change(s) will operate in the production FPKI as intended. When tests fail in the CITE, issues are identified and addressed without production FPKI impact. <br>
+
+## Terms and Conditions
```
Comment (lachellel) [inline _tools/cite-guide.md]: This isn't a legal agreement - probably should  use other words than "Terms and Conditions"

File: _tools/cite-guide.md [hunk 5]
```diff
@@ -0,0 +1,459 @@
...
+
+# Community Interoperability Test Environment (CITE) Participation Guidance
+
+Prepared By: The FPKI Technical Working Group (TWG)  <br>
+An FPKI Policy Authority Working Group
+
+Last Update: February 12th, 2019 <br>
+
+## Overview
+
```
Comment (lachellel) [inline _tools/cite-guide.md:14]: Need to have links to each section below for easy navigation.

File: _tools/cite-guide.md [hunk 6]
```diff
@@ -0,0 +1,459 @@
...
+{% include alert-warning.html content="CITE Test Common Policy and Test Federal Bridge, at a minimum, will provide 20% availability per month. CITE should not be used or relied upon for near-production availability." %} <br>
+
+## Technical Support Availability
+
+CITE Participants shall provide the FPKI Technical Working Group with email and phone information for at least two technical contactsto help coordinate any technical service issues. In lieu of providing individual names for technical POCs, CITE Participants may establish a group or other organizational-based email addresses for communications with the appropriate technical contacts. This information will only be made available (in a controlled manner) to CITE Participants, FPKI Applicants (if applicable), and vendors supporting the FPKI as needed during testing or troubleshooting. CITE Participants involved in scheduled testing shall provide the issuance, management, and troubleshooting necessary to help resolve any issues.
+
+{% include alert-warning.html content="CITE Participant technical support is only available for scheduled testing with any outage resolved on a best effort basis. " %} <br>
+
+## Test Websites
```
Comment (lachellel) [inline _tools/cite-guide.md:93]: If there is none - don't list at all.

File: _tools/cite-guide.md [hunk 7]
```diff
@@ -0,0 +1,459 @@
...
+### Policy Object Identifiers (OIDS) and Mapping Table
+
+The table below lists the current test to production OID equivalent used by the FPKIMA and CITE Participants.
+
+1. [Federal PKI Trust Infrascture Test OIDs](#federal-pki-trust-infrastructure-test-oids)
+2. [Federal Agency PKI Test OIDs](#federal-agency-pki-test-oids)
+3. [Federal Shared Service Provider (SSP) Test OIDs](#federal-shared-service-provider-ssp-test-oids)
+4. [Non-Federal Issuer (NFI) Test OIDs](#non-federal-issuer-nfi-test-oids)
+5. [Commercial PKI Bridge Test OIDs](#commercial-pki-bridge-test-oids)
+6. [Deprecated Test OIDs](#deprecated-test-oids)
```
Comment (lachellel) [inline _tools/cite-guide.md]: you had a question about deprecated test oids in your notices - if the deprecated test oids are not being used, remove from the list.  Too much information!

File: _tools/cite-guide.md [hunk 8]
```diff
@@ -0,0 +1,459 @@
...
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.15.31 | id-dhs-certpcy-rudimentary | 2.16.840.1.101.3.2.1.15.1 |
+| 2.16.840.1.101.3.2.1.15.32 | id-dhs-certpcy-basic | 2.16.840.1.101.3.2.1.15.2 |
+| 2.16.840.1.101.3.2.1.15.33 | id-dhs-certpcy-medium | 2.16.840.1.101.3.2.1.15.3 |
+| 2.16.840.1.101.3.2.1.15.34 | id-dhs-certpcy-high | 2.16.840.1.101.3.2.1.15.4 |
+| 2.16.840.1.101.3.2.1.15.35 | id-dhs-certpcy-mediumHardware | 2.16.840.1.101.3.2.1.15.5 |
+
+#### Department of Justice (DOJ)
```
Comment (lachellel) [inline _tools/cite-guide.md]: Justice OIDs should no longer be active?  No cross-certs in prod.

File: _tools/cite-guide.md [hunk 9]
```diff
@@ -0,0 +1,459 @@
...
+| 2.16.840.1.101.3.2.1.48.56 | treasury-level 2 | 2.16.840.1.101.3.2.1.5.3 |
+| 2.16.840.1.101.3.2.1.48.57 | treasury-level 3 | 2.16.840.1.101.3.2.1.5.4 |
+| 2.16.840.1.101.3.2.1.48.58 | treasury-level 4 | 2.16.840.1.101.3.2.1.5.5 |
+| 2.16.840.1.101.3.2.1.48.75 | Treasury Medium-Software | 2.16.840.1.101.3.2.1.5.7 |
+| 2.16.840.1.101.3.2.1.48.76 | Treasury Basic Org | 2.16.840.1.101.3.2.1.5.8 |
+| 2.16.840.1.101.3.2.1.48.111 | treasury-pivi-hardware | 2.16.840.1.101.3.2.1.5.10 |
+| 2.16.840.1.101.3.2.1.48.112 | treasury-pivi-cardAuth | 2.16.840.1.101.3.2.1.5.11 |
+| 2.16.840.1.101.3.2.1.48.113 | treasury-pivi-contentSigning | 2.16.840.1.101.3.2.1.5.12 |
+
+#### Federal Bureau of Investigation (FBI)
```
Comment (lachellel) [inline _tools/cite-guide.md]: FBI no longer active?  No cross-certs in prod?

File: _tools/cite-guide.md [hunk 10]
```diff
@@ -0,0 +1,459 @@
...
+| 2.16.840.1.101.3.2.1.48.251 | APL test lab Golden PIV-I Digital Signature | N/A |
+| 2.16.840.1.101.3.2.1.48.252 | APL test lab Golden PIV-I Content Signing | N/A |
+
+#### Government Printing Office (GPO)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.37 | id-gpo-medium | 2.16.840.1.101.3.2.1.17.1 |
+
+#### Internal Revenue Service (IRS)
```
Comment (lachellel) [inline _tools/cite-guide.md]: IRS no longer active?  no cross-certs in prod?

File: _tools/cite-guide.md [hunk 11]
```diff
@@ -0,0 +1,459 @@
...
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.74 | NASA | 1.3.6.1.4.1.71.1.1.103 |
+
+### Federal Shared Service Provider (SSP) Test OIDs
+
+See [Federal PKI Federal Common Policy](#federal-pki-federal-common-policy)
+
+### Non-Federal Issuer (NFI) Test OIDs
+
+#### Entrust Management Services
```
Comment (lachellel) [inline _tools/cite-guide.md]: Entrust **Managed** Services

File: _tools/cite-guide.md [hunk 12]
```diff
@@ -0,0 +1,459 @@
...
+| 2.16.840.1.114412.99.4.1.2 | DigiCert Level 1 Client Certificate - Enterprise | 2.16.840.1.114412.4.1.2 |
+| 2.16.840.1.114412.99.4.2 | DigiCert Level 2 Client Certificate - Basic | 2.16.840.1.114412.4.2 |
+| 2.16.840.1.114412.99.4.3.1 | DigiCert Level 3 Client Certificate - US -Medium | 2.16.840.1.114412.4.3.1 |  
+| 2.16.840.1.114412.99.4.3.2 | DigiCert Level 3 Client Certificate - CBP -Medium | 2.16.840.1.114412.4.3.2 |
+| 2.16.840.1.114412.99.4.4.1 | Digicert Level 4 Client Certificate - US - Hardware | 2.16.840.1.114412.4.4.1 |
+| 2.16.840.1.114412.99.4.4.2 | Digicert Level 4 Client Certificate - CBP -  Hardware | 2.16.840.1.114412.4.4.2 |
+| 2.16.840.1.114412.99.4.5.1 | Level 4 PIV-I - Hardware | 2.16.840.1.114412.4.5.1 |
+| 2.16.840.1.114412.99.4.5.2 | DigiCert Level 4 PIV-I - Card Authentication | 2.16.840.1.114412.4.5.2 |
+| 2.16.840.1.114412.99.4.5.3 | DigiCert Level 4 PIV-I - Content Signing | 2.16.840.1.114412.4.5.3 |
+| 2.16.840.1.114412.99.1.11 | DigiCert OV SSL | 2.16.840.1.114412.1.11 |
```
Comment (lachellel) [inline _tools/cite-guide.md]: We shouldn't have an OV SSL mapped anymore (pending March). ?

File: _tools/cite-guide.md [hunk 13]
```diff
@@ -0,0 +1,459 @@
...
+| 1.3.6.1.4.1.24019.1.1.1.108 | CertiPath IceCAP-cardAuth | 1.3.6.1.4.1.24019.1.1.1.8 |
+| 1.3.6.1.4.1.24019.1.1.1.109 | CertiPath IceCAP-contentSigning | 1.3.6.1.4.1.24019.1.1.1.9 |
+| 1.3.6.1.4.1.24019.1.1.1.110 | CertiPath variant medium Software | 1.3.6.1.4.1.24019.1.1.1.17 |
+| 1.3.6.1.4.1.24019.1.1.1.111 | CertiPath variant medium Hardware | 1.3.6.1.4.1.24019.1.1.1.18 |
+| 1.3.6.1.4.1.24019.1.1.1.112 | CertiPath variant high Hardware | 1.3.6.1.4.1.24019.1.1.1.19 |
+| 1.3.6.1.4.1.24019.1.1.1.113 | CertiPath variant medium CBP Software | 1.3.6.1.4.1.24019.1.1.1.20 |
+| 1.3.6.1.4.1.24019.1.1.1.114 | CertiPath variant medium CBP Hardware | 1.3.6.1.4.1.24019.1.1.1.21 |
+| 1.3.6.1.4.1.24019.1.1.1.115 | CertiPath variant high CBP Hardware | 1.3.6.1.4.1.24019.1.1.1.22 |
+
+### Deprecated Test OIDS
```
Comment (lachellel) [inline _tools/cite-guide.md]: I'd just remove all these deprecated test oids.  You can track somewhere for historical purposes but they are irrelevant to current?

**Comment by weirdscience** [comment]
Thank you @lachellel . I'll get started on the comments.

File: _tools/cite-guide.md (hunk 12 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 10 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 9 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 8 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 6 above)
Comment (weirdscience) [reply _tools/cite-guide.md:93]: I like to keep it as a placeholder if an affiliate wants to share one, we know where to put it.

File: _tools/cite-guide.md (hunk 1 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 2 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done

File: _tools/cite-guide.md (hunk 3 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Removed

File: _tools/cite-guide.md (hunk 4 above)
Comment (weirdscience) [reply _tools/cite-guide.md]: Done.

File: _tools/cite-guide.md (hunk 5 above)
Comment (weirdscience) [reply _tools/cite-guide.md:14]: Added.

**Comment by lachellel** [comment]
@weirdscience 

do you want a review and a merge?  (not sure if this PR is ready or still waiting for edits!)

**Comment by weirdscience** [comment]
@lachellel reviewed and looks good from here.

File: _tools/cite-guide.md [hunk 14]
```diff
@@ -0,0 +1,352 @@
...
+
+This guide is a practice guide for FPKI Partners who want to either become CITE participants or leverage CITE for FPKI testing.
+
+## Testing Use Cases
+
+The main purpose of CITE is interoperability and infrastructure testing of PKI components and Relying Party applications. Additional types of testing may be identified and conducted as necessary and to the extent supported by CITE Participants. CITE should not be used for system stress testing. Infrastructure testing ensures that upgrades, patches, policy changes, new products, and any other changes to the production FPKI do not adversely affect interoperability.
+
+Relying Party application testing ensures that application modules operate as intended. In addition, application testing ensures that the system performs as expected and can properly process transactions that rely on FPKI certificates.
+
+This document does not define how to perform testing in CITE. That is a responsibility of the CITE Participatns. Some examples of testing conducted in CITE include:
```
Comment (lachellel) [inline _tools/cite-guide.md]:
```suggestion
This document does not define how to perform testing in CITE. That is a responsibility of the CITE Participants. Some examples of testing conducted in CITE include:
```

File: _tools/cite-guide.md [hunk 15]
```diff
@@ -0,0 +1,352 @@
...
+1. Interoperability testing between cross-certified Certification Authorities (CAs);
+2. Transition testing to new algorithms (e.g., SHA-2, ECC);
+3. PIV and PIV-I card interoperability testing;
```
Comment (lachellel) [inline _tools/cite-guide.md]:
```suggestion
3. PIV and PIV-I credential interoperability testing;
```

**Comment by weirdscience** [comment]
@lachellel thank you for the updates. Looks good to merge.

**Pull request merged**